src/plugins/kibana/public/visualize/styles/main.less

```diff
@@ -2,21 +2,247 @@
 @import (reference) "~bootstrap/less/list-group";
 @import (reference) "~ui/styles/list-group-menu";
 
+@icon-background-color: #416376;
+
+ul.li-not-striped {
+  li:nth-child(odd) {
+    background-color: transparent !important;
+  }
+
+  li:nth-child(even) {
+    background-color: transparent !important;
+  }
+
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+}
+
+ul.li-striped {
+  li {
+    border: none;
+  }
+
+  li:nth-child(odd) {
+    background-color: @kibanaGray6;
+  }
+
+  li:nth-child(even) {
+    background-color: @white;
+  }
+}
+
+.step-1-wizard {
+  flex: 1;
+  background-color: @kibanaGray6;
+}
+
+.step-2-list {
+  border: none;
+  background-color: @kibanaGray6;
+  border-radius: 0;
+  padding-left: 5px;
+}
+
+.search-label {
+  padding-left: 5px;
+}
+
+.list-search-item {
+  border: none;
+  background-color: @kibanaGray6;
+  border-radius: 0;
+  padding: 0;
+}
+
+.saved-search-label {
+  padding-left: 10px;
+
+  label {
+    margin-bottom: 0;
+  }
+}
+
+.paginate-style {
+  padding-bottom: 1px;
+  background-color: @kibanaGray6;
+}
+
+.form-text {
+  color: @kibanaBlue2;
+}
+
+.link-color {
+  color: @kibanaBlue2;
+}
+
+.step-2-table {
+  margin-bottom: 0;
+  border-bottom: 1px solid @kibanaGray6;
+}
+
+.step-2-table-header {
+  padding: 16px !important;
+}
+
+.index-link {
+  text-decoration: @link-hover-decoration !important;
+  color: @kibanaBlue2;
+}
+
+.icon-color {
+  color: @icon-background-color;
+}
+
+.search-icon {
+  color: @kibanaGray3;
+  width: 30px !important;
+}
+
+.table-type-label {
+  color: @kibanaGray2 !important;
+  font-weight: normal;
+}
+
+.table-label {
+  color: @kibanaGray1 !important;
+  font-weight: normal;
+}
+
+/* saved_object_finder.html striped table colors */
+.table-striped > tbody > tr:nth-of-type(even) {
+  background-color: @kibanaGray6;
```

Comment:
Is there something more semantic we can use as the naming for all of the table styles that use `kibanaGray6`? Something they have in common?